packages/web/src/fixtures/dev-kit/hooks.ts

```diff
@@ -135,17 +135,17 @@ export const useCancelStaking = () => {
   return { cancel, isLoading, error }
 }
 
-export const useAssetStrength = (metricsAddress: string, marketAddress: string) => {
+export const useAssetStrength = (metricsAddress?: string, marketAddress?: string) => {
   const { data: metrics, error: metricsError } = useSWR<UnwrapFunc<typeof getLastAssetValueEachMetrics>, Error>(
-    SWRCachePath.getLastAssetValueEachMetrics(metricsAddress),
-    () => getLastAssetValueEachMetrics(metricsAddress),
+    SWRCachePath.getLastAssetValueEachMetrics(metricsAddress ? metricsAddress : ''),
```

Comment:
I'm not satisfied with the implementation here and would like some advice on how to handle undefined:pray: